src/octoprint/timelapse.py

```diff
@@ -853,6 +887,10 @@ def _render(self):
 			finally:
 				self._notify_callback("always", output)
 
+	@staticmethod
+	def _parsetime(hours, minutes, seconds):
```

Comment:
It doesn't really `parse` though but rather `convert`, right? 🤔 Or am I misunderstanding something here?